RadixWallet/Features/CreatePersona/Children/Introduction/IntroductionToPersonas+View.swift

```diff
@@ -18,35 +12,33 @@ extension IntroductionToPersonas {
 		}
 
 		public var body: some SwiftUI.View {
-			WithViewStore(
-				store,
-				observe: IntroductionToPersonas.ViewState.init(state:),
-				send: { .view($0) }
-			) { viewStore in
+			WithPerceptionTracking {
 				ScrollView {
 					VStack(spacing: .large2) {
-						introToPersona(with: viewStore)
+						introToPersona
 					}
 					.multilineTextAlignment(.center)
-					.padding(.horizontal, .large1)
+					.padding(.horizontal, .large2)
 					.padding(.bottom, .medium2)
 				}
 				.footer {
 					Button(L10n.CreatePersona.Introduction.continue) {
-						viewStore.send(.continueButtonTapped)
+						store.send(.view(.continueButtonTapped))
 					}
 					.buttonStyle(.primaryRectangular)
 				}
-				.onAppear { viewStore.send(.appeared) }
+				.onAppear { store.send(.view(.appeared)) }
 				.destinations(with: store)
 			}
 		}
 
 		@ViewBuilder
-		private func introToPersona(with viewStore: ViewStoreOf<IntroductionToPersonas>) -> some SwiftUI.View {
+		private var introToPersona: some SwiftUI.View {
```

Comment:
I would just put this stuff inline, or if you really want to extract it, bring the VStack along. ViewBuilder variables with multiple elements can cause surprises later.